### PR TITLE
Added text node + node expand bug fixed-ish

### PR DESCRIPTION
**Pull request by parkern342**
Added a text node to gimp frontend and overall backend.

Also fixed bug where a node would visually expand the text on an unexpanded node when a new node is added to the node view.
Now you just need to click anywhere to get rid of the bug.

### CODE REVIEW

File: backend/ontario/ontario/__init__.py [hunk 1]
```diff
@@ -393,6 +393,32 @@ def tileglass(self, width: int, height: int) -> "ImageBuilder":
...
+        node.set_property("font", font)
+        node.set_property("size", size)
+        node.set_property("color", Gegl.Color.new(color))
+        node.set_property("wrap_width", wrap_width)
+        node.set_property("wrap_height", wrap_height)
+        node.set_property("alignment", alignment)
+        node.set_property("vert_alignment", vert_alignment)
+
+        self.__nodes.append(node)
+        return node.get_property("width"), node.get_property("height")
```
Comment (notgull) [inline backend/ontario/ontario/__init__.py:420]: Is this an ImageBuilder?

Comment (parkern342) [reply backend/ontario/ontario/__init__.py:420]: This generates text as an image buffer and returns the width and height of the theoretical text so you can construct a shadow buffer with that information later.

Comment (notgull) [reply backend/ontario/ontario/__init__.py:420]: The type information here says that it returns an `ImageBuilder` though...

https://github.com/TeamPictonode/pictonode/blob/2452364a03b6c22abe91ee7136877393fd0af5d5/backend/ontario/ontario/__init__.py#L404

Comment (parkern342) [reply backend/ontario/ontario/__init__.py:420]: I see, that was just copied and pasted, will fix. Thanks!

**Comment by Stehfyn** [comment]
Nice work!

**Pull request merged**